go/vt/wrangler/testlib/init_shard_master_test.go

```diff
@@ -238,7 +297,9 @@ func TestInitMasterShardOneSlaveFails(t *testing.T) {
 	_, err := ts.UpdateShardFields(ctx, master.Tablet.Keyspace, master.Tablet.Shard, func(si *topo.ShardInfo) error {
 		// note it's OK to retry this and increment multiple times,
 		// we just want it to be different
-		si.MasterAlias.Uid++
+		if si.MasterAlias != nil {
```

Comment:
Why is it ok for this to be `nil` now?

Reply:
I must have changed this before I added the "wait for shard record to be updated with MasterAlias". I'll revert it.